prebid-mobile/modules/rendering/ios-sdk-integration-pb.md

```diff
@@ -2,14 +2,56 @@
 
 layout: page_v2
 title: Custom or No mediation
-description: Integration of Prebid SDK without Primary Ad Server SDK
+description: How to integrate the Prebid SDK without a primary Ad Server SDK
 sidebarType: 2
 
 ---
 
 # Custom Bidding Integration
 {:.no_toc}
 
+## Before you start
```

Comment:
I think this section should be a short "intro" to what's going to happen.

1. You're going to initialize the SDK
2. You're going to set up AdUnits of various types
3. You're going to initialize the auction
4. Finally, you're going to render the results.

Then add a new section for "Initialization"

Reply:
Definitely progress here, but I still question the "Transport API" thing. IMO, the sections should be: Initialize the SDK, Set Up Adunits, Initialize the Auction, and Render Results.